src/calendar/header/index.js

```diff
@@ -113,6 +113,10 @@ class CalendarHeader extends Component {
         dayStyle.push(this.style.disabledDayHeader);
       }
 
+      if (this.style[`day${idx}Text`]) {
+        dayStyle.push(this.style[`day${idx}Text`]);
```

Comment:
I'll prefer a more descriptive name, like: 'dayTextAtIndex${idx}'. Also please add this feature to the readme nad the demo screen so the users can be aware of it.